data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -345,23 +371,28 @@ then
         --volume "$PWD:/assets:z" \
         --volume /etc/kubernetes:/etc/kubernetes:z \
         "${CLUSTER_BOOTSTRAP_IMAGE}" \
-        start --tear-down-early=false --asset-dir=/assets --required-pods="openshift-kube-apiserver/kube-apiserver,openshift-kube-scheduler/openshift-kube-scheduler,openshift-kube-controller-manager/kube-controller-manager,openshift-cluster-version/cluster-version-operator"
+        start --tear-down-early=false --asset-dir=/assets --required-pods="$REQUIRED_PODS"
+
     touch cb-bootstrap.done
 fi
 
-rm --force /etc/kubernetes/manifests/machineconfigoperator-bootstrap-pod.yaml
-
-if [ ! -z "$CLUSTER_ETCD_OPERATOR_IMAGE" ]
+if [ "$BOOTSTRAP_INPLACE" = true ]
 then
-	echo "Waiting for CEO to finish..."
-	bootkube_podman_run \
-		--volume "$PWD:/assets:z" \
-		"${CLUSTER_ETCD_OPERATOR_IMAGE}" \
-		/usr/bin/cluster-etcd-operator \
-			wait-for-ceo \
-			--kubeconfig /assets/auth/kubeconfig
+    . /usr/local/bin/bootstrap-in-place.sh "${CLUSTER_BOOTSTRAP_IMAGE}"
+else
+    rm --force /etc/kubernetes/manifests/machineconfigoperator-bootstrap-pod.yaml
+
+    if [ ! -z "$CLUSTER_ETCD_OPERATOR_IMAGE" ]
+    then
+        echo "Waiting for CEO to finish..."
+        bootkube_podman_run \
```

Comment:
nit: this could be flattened out of the if/then starting on 379... if bootstrap_inplace=false and ! -z cluster_edcd_operator_image; then...

Reply:
This will change the behavior of non bootstrap-in-place installation.
`rm --force /etc/kubernetes/manifests/machineconfigoperator-bootstrap-pod.yaml`
Doesn't depend on CLUSTER_ETCD_OPERATOR_IMAGE env.

Also, the machineconfigoperator-bootstrap-pod. is required for bootstrap-in-place.sh so it shouldn't be removed in case "$BOOTSTRAP_INPLACE" = true

Reply:
Allow me to clarify.... keep:
```
if [ "$BOOTSTRAP_INPLACE" = true ]
then 
    . /usr/local/bin/bootstrap-in-place.sh "${CLUSTER_BOOTSTRAP_IMAGE}"
else
    rm --force /etc/kubernetes/manifests/machineconfigoperator-bootstrap-pod.yaml
fi
```
and the flatten the rest:
```
if [ ! -z "$CLUSTER_ETCD_OPERATOR_IMAGE" ] && [ ! "$BOOTSTRAP_INPLACE" = true ]
then
        echo "Waiting for CEO to finish..."
```

Reply:
I don't like the first part of that. Combining the call to `bootstrap-in-place.sh` with the remocal of the MCO pod makes it seem like they are more closely related than they are.

Reply:
Are you talking about switching that to:
```
if [ "$BOOTSTRAP_INPLACE" = true ]
then 
    . /usr/local/bin/bootstrap-in-place.sh "${CLUSTER_BOOTSTRAP_IMAGE}"
fi

if [ ! "$BOOTSTRAP_INPLACE" = true ]
then 
    rm --force /etc/kubernetes/manifests/machineconfigoperator-bootstrap-pod.yaml
fi
```